### DISCUSSION TITLE
How to sketch (model) this basket on FDS

### DISCUSSION THREAD

**Discussion by ahmedahmed2056** [firemodels/fds#13055] [General]
![image](https://github.com/firemodels/fds/assets/165714562/dca6774e-b41d-4cf8-8fc4-b152b5e4f350)

**### It is 20 cm height**

**Comment by drjfloyd** [comment]
No one knows what you are trying to do to be able to make any suggestions. 

Are you modeling a wildland fire on a 10 km x 10 km domain with 10 m grid cells and the basket is sitting on the ground somewhere? In that case a 20-40 cm basket the best answer is probably to not bother modeling the basket.

Are you doing a DNS simulation with 0.2 mm grid cells and have some flame impinging on the mesh? In that case you can resolve the wires on the grid and you should create OBST and/or GEOM to represent the detailed geometry of the basket.

**Reply by ahmedahmed2056** [reply]
Thanks for your reply. It is a lab experiment of a basket having Pinus pinaster needles in a basket. So here I am interested in the combustion of such fuel inside the basket. I will use LES.
